requirements.txt

```diff
@@ -24,5 +24,7 @@ pandas>=1.4.1,<2.0.0
 pydantic==1.9.1
 discord.py==1.7.3
 pytest==7.2.0
-django-directory
+# django-directory
+# django-dirctory is currently incompatible with Django 4.0+, PR is pending to fix django-directory upstream
+git+https://github.com/cemathey/django-directory@f51ee1e8dc50edf453fee4a0d9631c0e46fe9433
```

Comment:
do we really need this? And if so, for what 😅

Reply:
It's used for the api/logs/ endpoint that lets people browse logs through the UI.